api/authentication/models.py

```diff
@@ -36,21 +36,27 @@ def _create_user(self, email, password, **extra_fields):
         return user
 
     def create_user(self, email, password=None, **extra_fields):
-        extra_fields.setdefault('is_staff', False)
-        extra_fields.setdefault('is_superuser', False)
+        extra_fields.setdefault('is_moderator', False)
+        extra_fields.setdefault('is_admin', False)
         extra_fields.setdefault('is_active', False)
 
         return self._create_user(email, password, **extra_fields)
 
-    def create_superuser(self, email, password, **extra_fields):
-        extra_fields.setdefault('is_staff', True)
-        extra_fields.setdefault('is_superuser', True)
+    def create_admin(self, email, password, **extra_fields):
+        extra_fields.setdefault('is_admin', True)
         extra_fields.setdefault('is_active', True)
 
-        if extra_fields.get('is_staff') is not True:
-            raise ValueError('Superuser must have is_staff=True.')
-        if extra_fields.get('is_superuser') is not True:
-            raise ValueError('Superuser must have is_superuser=True.')
+        if extra_fields.get('is_admin') is not True:
+            raise ValueError('Superuser must have is_admin=True.')
+
+        return self._create_user(email, password, **extra_fields)
+
+    def create_moder(self, email, password, **extra_fields):
```

Comment:
I think method will be the same as option in user model `is_moderator`. Please rename to `create_moderator`, it will be more simple for understanding.